File: week_1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Get positive integer between 1 and 8 (inclusive) from user
    int stored;

    do
    {
        stored = get_int("Height: ");
    }
    while (stored < 1 || stored > 8);

    for (int i = 0; i < stored; i++)
    {
        // For-Loop to add spaces
        for (int k = stored - i; k > 1; k--)
        {
            printf(" ");
        }
        // For-Loop to add hashes
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }
        printf("\n");
    }

}

File: week_6/sentimental-mario-more/mario.py
from cs50 import get_int

# Get positive integer between 1 and 8 inclusive
while True:
    # Ask for input
    height = get_int("Height: ")
    # Check condition
    if height > 0 and height < 9:
        break

# Draw the pyramide
for i in range(1, height + 1):
    # For loop to add spaces
    for j in range(height - i):
        print(" ", end="")

    # For loop to add hashes
    for k in range(i):
        print("#", end="")

    # Add space between blocks
    print("  ", end="")

    # For loop to add hashes
    for l in range(i):
        print("#", end="")

    print()

